tests/visual/pdf2png.js

```diff
@@ -1,6 +1,6 @@
 import Canvas from 'canvas';
 import { strict as assert } from 'assert';
-import pdfjsLib from 'pdfjs-dist/es5/build/pdf';
+import * as pdfjsLib from 'pdfjs-dist/es5/build/pdf';
```

Comment:
Why this change?

Reply:
The original didn't work for me (`pdfjsLib` was undefined). This fixes it for me. If it's not an issue for you/others, I can easily remove or revert this and just patch it locally for myself. On the other hand, if it still works for you/others, this one works for me as well, so might be nice to keep it.

Reply:
No problem. Let's keep this change